modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -8,122 +8,170 @@ CREATE ROLE regularUsers;
 ////
 
 :page-role: enterprise-edition aura-db-business-critical aura-db-dedicated
+
 [[property-based-access-control]]
 = Property-based access control
 
-It is possible to create read privileges that are based on properties of nodes.
+Property-based access control grants permissions to users to read node properties based on property/value conditions.
 Each property-based privilege can only be restricted by a single property.
-To specify the property/value conditions of the read privilege the `pattern` syntax described below is used,
-for more information about read privilege syntax see xref:authentication-authorization/privileges-reads.adoc[read privilege] page.
+For information about read privileges and their syntax, see xref:authentication-authorization/privileges-reads.adoc[Read privileges].
+
+[IMPORTANT]
+====
+When using property-based access control, ensure the property used for the rule cannot be modified.
+Users who can change this property can affect the granted property-based privileges.
+====
+
+
+== Syntax
+
+To specify the property/value conditions of the read privilege, you can use the following syntax:
+
+[source, syntax, role="noheader"]
+----
+GRANT|DENY [IMMUTABLE] READ|TRAVERSE|MATCH
+  ON { HOME GRAPH | GRAPH[S] { * | name[, ...] } }
+    [
+      ELEMENT[S] { * | label-or-rel-type[, ...] }
+      | NODE[S] { * | label[, ...] }
+      | RELATIONSHIP[S] { * | rel-type[, ...] }
+      | FOR {
+  
+([var][:label["|" ...]] "{" property: value "}") 
+  | (var[:label["|" ...]]) 
+      WHERE [NOT] var.property { { = | <> | > | >= | < | <= } value | IS NULL | IS NOT NULL | IN { "["[value[, ...]]"]" | listParam } }
+  | (var[:label["|" ...]] 
+WHERE [NOT] var.property { { = | <> | > | >= | < | <= } value | IS NULL | IS NOT NULL | IN { "["[value[, ...]]"]" | listParam } } ) 
+}
+
+  TO role[, ...]
+----
```

Comment:
@phil198 hi,

Could you please say if this syntax is correct for all possible property-based privileges?